en/getting_started/rc_transmitter_receiver.md

```diff
@@ -73,40 +72,43 @@ In addition, receivers have to be [compatible with PX4](#compatible_receivers) a
 Compatible radio systems are often sold together.
 For example, [FrSky Taranis X9D and FrSky X8R](https://hobbyking.com/en_us/frsky-2-4ghz-accst-taranis-x9d-plus-and-x8r-combo-digital-telemetry-radio-system-mode-2.html?___store=en_us) are a popular combination.
 
-
 ### Transmitter/Receiver Pairs
 
-One of the most popular RC units is the *FrSky Taranis X9D*.
-It has an internal transmitter module can be used with the recommended *FrSky X4R-SB* (S-BUS, low delay) or *X4R* (PPM-Sum, legacy) receivers out of the box.
-It also has a custom radio transmitter module slot and customizable open source OpenTX Firmware. 
+One of the most popular RC units is the _FrSky Taranis X9D_.
+It has an internal transmitter module can be used with the recommended _FrSky X4R-SB_ (S-BUS, low delay) or _X4R_ (PPM-Sum, legacy) receivers out of the box.
+It also has a custom radio transmitter module slot and customizable open source OpenTX Firmware.
 
 ::: info
 This remote control unit can display vehicle telemetry when used with [FrSky](../peripherals/frsky_telemetry.md) or [TBS Crossfire](../telemetry/crsf_telemetry.md) radio modules.
 :::
 
 Other popular transmitter/receiver pairs
 
-* Turnigy remote using, for example, the FrSky transmitter/receiver modules.
-* Futaba Transmitters and compatible Futaba S-Bus receivers.
-* Long range ~900MHz, low latency: "Team Black Sheep Crossfire" or "Crossfire Micro" set with a compatible remote (e.g. Taranis)
-* Long Range ~433MHz: ImmersionRC EzUHF set with a compatible remote (e.g. Taranis)
-
+- Turnigy remote using, for example, the FrSky transmitter/receiver modules.
+- Futaba Transmitters and compatible Futaba S-Bus receivers.
+- Long range ~900MHz, low latency: "Team Black Sheep Crossfire" or "Crossfire Micro" set with a compatible remote (e.g. Taranis)
+- Long Range ~433MHz: ImmersionRC EzUHF set with a compatible remote (e.g. Taranis)
 
-<a id="compatible_receivers"></a>
-
-### PX4-Compatible Receivers
+### PX4-Compatible Receivers {#compatible_receivers}
 
 In addition to the transmitter/receiver pairs being compatible, the receiver must also be compatible with PX4 and the flight controller hardware.
 
-*PX4* and *Pixhawk* have been validated with:
+_PX4_ and _Pixhawk_ have been validated with:
 
 - All Spektrum DSM RC receivers
-- All Futaba S.BUS and S.BUS2 RC receivers
-- All FrSky PPM and S.Bus models
-- Graupner HoTT
-- All PPM models from other manufacturers
-- TBS Crossfire/Express LRS Receivers using [CRSF Telemetry](../telemetry/crsf_telemetry.md) (UART connection). 
+- All Futaba S.BUS and S.BUS2 RC receivers.
+- All FrSky S.BUS and PPM models

```

Comment:
Would this mean S.BUS and S.BUS2 (i.e. is the situation the same for both)?

Reply:
I think SBUS 1 vs. 2 is a distinction from the transition time in 2013. We might mention that it is backward compatible in one place, but I expect most people are not explicitly looking to fit 10+ year old devices.